### PR TITLE
Moes Door sensor /  ZSS-JM-GWM-C-MS / update tuya.ts

### PR DESCRIPTION
**Pull request by mainmind83**
New Moes device ZSS-JM-GWM-C-MS

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -565,6 +565,7 @@ const definitions: Definition[] = [
             tuya.whitelabel('TuYa', 'MC500A', 'Door sensor', ['_TZ3000_2mbfxlzr']),
             tuya.whitelabel('TuYa', '19DZT', 'Door sensor', ['_TZ3000_n2egfsli']),
             tuya.whitelabel('TuYa', 'DS04', 'Door sensor', ['_TZ3000_yfekcy3n']),
+            tuya.whitelabel('Moes', 'Smart Door and Window Sensor', 'Door sensor', ['_TZ3000_decxrtwa']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:568]: Please use a proper model number he, something that is written on the device/box.

Comment (mainmind83) [reply src/devices/tuya.ts]: Sorry, my fault... added.

**Review by mainmind83 — COMMENTED** [review]
Added model ID

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**